core/src/main/scala/org/apache/spark/storage/BlockObjectWriter.scala

```diff
@@ -20,10 +20,17 @@ package org.apache.spark.storage
 import java.io.{BufferedOutputStream, FileOutputStream, File, OutputStream}
 import java.nio.channels.FileChannel
 
-import org.apache.spark.Logging
-import org.apache.spark.serializer.{SerializationStream, Serializer}
-import org.apache.spark.executor.ShuffleWriteMetrics
+import org.apache.hadoop.mapreduce.security.TokenCache
 
+import com.intel.chimera.{CipherSuite, CryptoOutputStream, CryptoCodec}
```

Comment:
nit: should be grouped with the `org.apache.hadoop` imports. See pattern in other files.